Doc/library/hashlib.rst

```diff
@@ -694,7 +694,7 @@ implementation, extension code, and this documentation:
 
    You should have received a copy of the CC0 Public Domain Dedication along
    with this software. If not, see
-   http://creativecommons.org/publicdomain/zero/1.0/.
+   https://creativecommons.org/publicdomain/zero/1.0/.
```

Comment:
This text is  a license. Is it ok to modify a license?

Reply:
I'm not sure about that. Should I ask this on the bug?

Reply:
I think it's ok. I take the responsability for this change ;-)

Reply:
Alright thanks :)